cli/main.py

```diff
@@ -1,3 +1,19 @@
+"""
```

Comment:
I respect the communism, but the comment should be after `#!/usr/bin/env python3`. Vim with flake8 should have told you that!

Reply:
I didn't use vim, I used a script to prepend the license header. If you see anymore, lmk.

Reply:
Just to nit-pick, I think this looks better / more convention with two new lines after the license text.